### DISCUSSION TITLE
Error in the backend witness.go file

### DISCUSSION THREAD

**Discussion by Ybittan** [Consensys/gnark#370] [Q&A] [answered]
Hello,

I am trying to compile my project, but am running into this. Any thoughts? 

![Screenshot_11](https://user-images.githubusercontent.com/31249213/190200638-cd6e24c1-1b1b-4edd-91e5-6d0d66bdd515.png)

**Comment by ivokub** [comment, marked as answer]
Recently, gnark-crypto changed the ecc.ID interface. You can get the bit-length of the modulus by:
```
w.curveID.ScalarField().BitLen()
```
